packages/api/pages/api/languages/[code]/verses/[verseId]/notes.ts

```diff
@@ -0,0 +1,57 @@
+import { GetVerseTranslatorNotesResponseBody } from '@translation/api-types';
+import createRoute from '../../../../../../shared/Route';
+import { Prisma, client } from '../../../../../../shared/db';
+
+export default createRoute<{ code: string; verseId: string }>()
+  .get<void, GetVerseTranslatorNotesResponseBody>({
+    async handler(req, res) {
+      const language = await client.language.findUnique({
+        where: {
+          code: req.query.code,
+        },
+      });
+      if (!language) {
+        return res.notFound();
+      }
+      const verse = await client.verse.findUnique({
+        where: {
+          id: req.query.verseId,
+        },
+        include: {
+          words: {
+            orderBy: {
+              id: Prisma.SortOrder.asc,
+            },
+          },
+        },
+      });
+
+      if (verse) {
+        const response: GetVerseTranslatorNotesResponseBody = { data: {} };
+        for (const word of verse.words) {
+          const note = await client.translatorNote.findUnique({
+            where: {
```

Comment:
Just like I did for loading verse glosses, can we write a custom SQL query that loads these notes all in one go rather than using a for loop? That will be far more performant if we only have one trip to the database rather than one for each word in the verse.